R/conclass.R

```diff
@@ -224,11 +269,11 @@ Conos <- R6::R6Class("Conos", lock_objects=FALSE,
         }
 
         
-        if(verbose) cat(".")
-        return(data.frame('mA.lab'=rownames(mnn)[mnn@i+1],'mB.lab'=colnames(mnn)[mnn@j+1],'w'=mnn@x,stringsAsFactors=FALSE))
+        if(verbose) message(".")
```

Comment:
Replacing `cat` with messages does not exactly work here, as these `cat`s are supposed to print in one lime. We actually should replace `cat(".")` with using a `lapply` with progress bar.

Reply:
> We actually should replace cat(".") with using a lapply with progress bar.

Good idea, will do.

Reply:
I've reverted back to `cat(".")` for now. We can add more going forward.